cohort-engine-api-web/src/test/java/com/ibm/cohort/engine/api/service/CohortEngineRestHandlerTest.java

```diff
@@ -758,6 +758,80 @@ public void testCohortEvaluation() throws Exception {
 		ServiceBaseUtility.apiCleanup(Mockito.any(), Mockito.eq(MethodNames.EVALUATE_COHORT.getName()));
 	}
 
+	@PrepareForTest({ Response.class, FHIRRestUtils.class })
+	@Test
+	public void testCohortMultipleDependencies() throws Exception {
```

Comment:
Was this test added to account for the multiple read issue? Just want to make sure we have coverage for the future.

Reply:
Yeah, this test makes sure the input stream isn't already "spent", and in master it fails.